generic/tests/netperf.py

```diff
@@ -130,6 +130,13 @@ def mtu_set(mtu):
         if netdst in br_in_use:
             ifaces_in_use = host_bridges.list_iface()
             target_ifaces = list(ifaces_in_use + br_in_use)
+        if params.get("netdst_nic1") in process.system_output(
+                "ovs-vsctl list-br", ignore_status=True, shell=True).decode():
+            ovs_list = "ovs-vsctl list-ports %s" % params["netdst_nic1"]
+            ovs_port = process.system_output(ovs_list, ignore_status=True,
```

Comment:
This `ignore_status=True` might be removed, since here it has to be an ovs environment and you have to get the right `ovs_port` with the command.

Reply:
Yes, I will fix it.